config/locales/en/apply/nonprofit.yml

```diff
@@ -16,10 +16,9 @@ en:
         Applications for Blueprint are closed for this semester. Sign up to be notified when they reopen!
       services:
         web_application_development: >
-          Our platform of choice, web applications leverage the power of the internet to expand reach and
-          increase communication.
+          Web applications let your users access information from their computer or on a browser. This might be better for applications that need to be publicly available or ones that help facilitate management tasks.
```

Comment:
First sentence sounds a bit redundant. Maybe try, "Web applications let your users access information from any browser. This might be better for applications that need to be publicly available or ones that help facilitate task management."